README.md

```diff
@@ -14,6 +21,6 @@ Other packages that process HTML files, e.g., `blaze-html-templates` (the defaul
 
 ## Top level head and body tags
 
-You can use head and body tags as usual.
+You can use head and body tags as usual with `svelte:compiler`.
```

Comment:
This section is only for `svelte:compiler` because the other one doesn't do anything with HTML (it uses `static-html` for head and body tags).